tests/test_auth.py

```diff
@@ -83,3 +84,13 @@ def test_capture_logger_output(self) -> None:
             logger.error("This is an error message")
 
         self.assertEqual(output.getvalue(), "This is an info message\nThis is an error message\n")
+
+    def test_whoami_raises_unauthenticated_error(self) -> None:
+        with self.assertRaises(UnauthenticatedError):
+            kagglehub.whoami()
+
+    def test_whoami_success(self) -> None:
+        with mock.patch("builtins.input") as mock_input:
+            mock_input.side_effect = ["lastplacelarry", "some-key"]
+            kagglehub.login(validate_credentials=False)
+            kagglehub.whoami()
```

Comment:
can you assert on the output of whoami to make sure the username matches `lastplacelarry`?